sim/bulb3dtest.py

```diff
@@ -1,16 +1,18 @@
 import params
-
-
 import runsim
 import odors
+import sys
 from math import sqrt
 
-params.filename = 'bulb3dtest'
-params.tstop = 1050
-params.coreneuron = False
-params.gpu = False
+params.tstop = int(sys.argv[-5])
+params.coreneuron = bool(int(sys.argv[-4]))
+params.gpu = bool(int(sys.argv[-3]))
+params.filemode = bool(int(sys.argv[-2]))
+params.filename = sys.argv[-1]
```

Comment:
Just a nitpick: 1. if args are not provided, we should print error message? (2. otherwise. args can be optional then we take previously mentioned values as default)

Reply:
I've added an argument parser with all the variables as optional ones and (some reasonable) default values. Let me know what you think. I also tested this and it works with python 2